actix/src/sec-6-sync-arbiter.md

```diff
@@ -1 +1,66 @@
-[WIP] 
\ No newline at end of file
+# SyncArbiter
+
+When you normally run Actors, there are multiple Actors sharing a single
+thread and event loop in the System. However for CPU bound workloads, or
+highly concurrent workloads, you may wish to have a single Actor able
```

Comment:
>you may wish to have a single Actor able

You may wish to offload Actor's work onto another threads

Reply:
Sorry, I don't think this comment adds clarity, I find the wording confusing. I have however tweaked the section you have mentioned here so you may find it better now.